app/api/schema/events.py

```diff
@@ -7,7 +7,8 @@
 from app.models.event import Event
 from app.api.helpers.exceptions import UnprocessableEntity
 from app.api.helpers.utilities import dasherize
-
+from sqlalchemy.orm.exc import NoResultFound
```

Comment:
This should come on top. 
1. Write line 1 and 5 together
2. then 10th and 11th below current 4th line. Since these are package level imports.

Reply:
Okay! I'll take care of these things from the next time.